src/common/SPELLS/warlock.js

```diff
@@ -296,6 +296,12 @@ export default {
     name: 'Wracking Brilliance',
     icon: 'spell_shadow_felmending',
   },
+  INEVITABLE_DEMISE_BUFF: {
```

Comment:
I forgot to mention this on the other PR, but wouldn't it be easier to put these into bfa/azeritetraits/warlock.js?

Reply:
Since Sudden Onset and Inevitable Demise are last two traits that are meaningful to implement (Deathbloom is pretty much dead), I'd prefer if you leave it *for now*, and after both are merged, I'll move all of the buffs into azeritetraits folder. Just so that it doesn't cause any conflicts etc.